yarn.lock

```diff
@@ -18,40 +18,40 @@
     "@jridgewell/gen-mapping" "^0.3.0"
     "@jridgewell/trace-mapping" "^0.3.9"
 
-"@axonivy/config-editor@~11.2.0-next.28":
```

Comment:
Do we need to have this file to be in the repository? Seems to be house keeping for yarn.

Reply:
yes, we want to have that file in the repo to get consistent installs across machines. see https://classic.yarnpkg.com/lang/en/docs/yarn-lock/

yarn.lock has changed slightly as I increased our own dependencies like @axonivy/config-editor and added new ones. if no dependencies are touched, it does not change.